inference-engine/src/low_precision_transformations/src/concat_multi_channels.cpp

```diff
@@ -178,7 +178,7 @@ bool ConcatMultiChannelsTransformation::transform(TransformationContext& context
             // for intermediate layers we should get Dq operations to be inserted between layer and child
             assert(dequantizationsToConcatenate.size() == 1ul);
             const size_t sourceOutputIdx = NetworkHelper::getParentOutputIndex(layer, child);
-            if (layer->get_input_shape(0)[1] != layer->get_output_shape(sourceOutputIdx)[1]) {
+            if (layer->get_input_partial_shape(0)[1] != layer->get_output_partial_shape(sourceOutputIdx)[1]) {
```

Comment:
are we sure rank is known here?

Reply:
This code will be executed for layers with `rank >= 2`. I added additional [check](https://github.com/openvinotoolkit/openvino/pull/6129/files#diff-73162195409816f357c4d1ceb96861614cae576a801c0e8519eb3f2dfffd5ff8R48), thank you! (subgraph.cpp methods fill a collection with layers for Concat & ConcatMultiChannelsTransformation)